frontend/src/metabase/components/IconWrapper/IconWrapper.jsx

```diff
@@ -1,18 +1,11 @@
 // eslint-disable-next-line no-restricted-imports
 import styled from "@emotion/styled";
 
-import { color } from "metabase/lib/colors";
-
 const IconWrapper = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  border-radius: ${props => props.borderRadius};
+  border-radius: ${({ borderRadius = 6 }) => borderRadius};
 `;
 
-IconWrapper.defaultProps = {
-  borderRadius: 6,
-  bg: color("bg-medium"),
```

Comment:
Applying `bg` to `styled.div` doesn't make sense. I'm pretty sure this is unused.